### DISCUSSION TITLE
Leave Default Device Icon

### DISCUSSION THREAD

**Discussion by bradcla** [andrew-codechimp/HA-Battery-Notes#799] [Ideas]
The add-in changes the default icon for a device from the previous set one to the Battery Notes icon. The add-in should not change the icon, leave the default.

**Comment by andrew-codechimp** [comment]
I'd like to be able to do this but HA has no way of specifying the primary integration. Please upvote this feature request. 

https://community.home-assistant.io/t/manage-primary-integration-for-devices/534948?u=codechimp

**Comment by andrew-codechimp** [comment]
To close out this, it appears the latest HA 2024.9 corrects this.
I have read for months on how you should declare secondary integrations for a device but never could get it working, now it magically has on both my prod and test systems.
